composables/drop/useGenerativeDropMint.ts

```diff
@@ -71,7 +69,7 @@ export default ({
   )
 
   const mintedCount = computed(() =>
-    Math.min(mintedDropCount.value, maxCount.value),
+    Math.min(collectionData.value.collectionEntity?.nftCount, maxCount.value),
```

Comment:
data from indexer

Reply:
is  `https://fxart.kodadot.workers.dev/drops/split/status` updated or should I base on ` https://fxart.kodadot.workers.dev/drops/split` ?

Reply:
It is another api side issue : )
You can see the minted data is zero for the opening drop 
<img width="935" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/a435e0a3-0218-4b01-92b5-3468e1fee45f">

Reply:
oh alright.. should I close this PR untill fix on API side?

Reply:
> oh alright.. should I close this PR untill fix on API side?

I'm not sure if we should hack on the frontend side
cc @vikiival  @preschian

Reply:
@preschian can you please chcek?

Reply:
> is https://fxart.kodadot.workers.dev/drops/split/status updated or should I base on https://fxart.kodadot.workers.dev/drops/split ?

oooh, I think I can adjust this on the worker's side. get `max || claim != 0`. currently, it only gets from `claim != 0` https://github.com/kodadot/private-workers/blob/ec6f6a5f88aa0c608541760ca6cb09a68a19ac83/fxart/src/routes/drops.ts#L138-L142

Since we use `max === minted` for the ended status

Reply:
https://github.com/kodadot/private-workers/pull/67